CONTRIBUTING.adoc

```diff
@@ -493,6 +494,14 @@ These will be automatically retried, but if you're in a hurry Michael has permis
 Nondeterministic failures are very annoying.
 Michael also has permissions to restart failed builds.
 
+==== Cicero
+
+[Cicero](https://github.com/input-output-hk/cicero) is our new CI and automation system.
+It builds all the same jobs as Hydra by reading `ci.nix`.
+
+For each job, it we generate a [Tullia](https://github.com/input-output-hk/tullia) task.
+Those can be run locally and on Cicero. Run `tullia list` to see all available tasks and `tullia run $job` to run it.
```

Comment:
We probably shouldn't say this unless we actually have `tullia` in the shell!

Reply:
@Pacman99 can I push that onto you? :grin:

Reply:
done